heroesfighter/src/Components/Card.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import contactCard from './ContactCard.jsx';
+import './Card.css'
+
+class Card extends React.Component{
+    render () {
+        
+        return(
+
+            <div className = 'Cards'>
+                 
+                <div className = 'Header'>
+                    <h2 className = 'Name'>
+                        {contactCard.name}
+                    </h2>
+                    <h3 className = 'Power'>
+                       POWER:<br/>
+                       <img src='https://cdn2.iconfinder.com/data/icons/superhero-neon-circle/64/10-superhero-512.png' /><br/>  
+                       {contactCard.powerstats.power}
```

Comment:
This row should only be contained in **h3** for great semantic.
**img** could be in a **figure** tag, "Power:" in a **span** tag and the all maybe in a **p** tag (look for the best solution)